packages/api-client/src/client/types.ts

```diff
@@ -66,5 +66,30 @@ export interface ApiFacets {
 
 export interface ApiQuery<T> {
     total_count?: number;
-    results: T[];
+    records: T[];
+}
+
+export enum ExportCatalogFormat {
+    CSV = 'csv',
+    JSON = 'json',
+    XLSX = 'xlsx',
+    RDF = 'rdf',
+    TTL = 'ttl',
+    DATA_SJON = 'data.json',
+    RSS = 'rss',
+    DCAT = 'dcat',
```

Comment:
Is there a reason to not list `dcat_ap_*` format ?

Reply:
I think it was added just a few days ago, now it appears in the doc

Reply:
Oh ok. Do you know if more format are gonna be added often ? Maybe a fixed enum isn't the best idea if format are gonna be added quite often.

Reply:
It was in 2.1 from the start, just not in the doc, it probably was a mistake. These formats are a bit special because they are activated per-domain, so they are not available everywhere (but the API Client has no way to know that), this is probably why the doc didn't mention them at first. But yes we could double-check if formats can be added in a stable version (after all, it would be a non-breaking change), maybe we shouldn't use an enum indeed